plugins/woocommerce-blocks/assets/js/blocks/product-filters/editor.scss

```
@@ -0,0 +1,46 @@
+$blue: #3858e9;
```

We should use color variable from Core here instead of only using fixed value.

```suggestion
$blue: var(--wp-components-color-accent, var(--wp-admin-theme-color, #3858e9));
```